characterisation/exceptions/HO200117.test.ts

```diff
@@ -0,0 +1,40 @@
+import World from "../../utils/world"
+import { processPhase2Message } from "../helpers/processMessage"
+import { reasonPath } from "../helpers/errorPaths"
+import generatePhase2Message from "../helpers/generatePhase2Message"
+import MessageType from "../types/MessageType"
+
+jest.setTimeout(30000)
+
+describe.ifPhase2("HO200117", () => {
+  afterAll(async () => {
+    await new World({}).db.closeConnection()
+  })
+
+  it("creates a HO200117 exception when there are more than 10 recordable results (there are 11)", async () => {
```

Comment:
Don't think it's necessary as it's clear from the test.

```suggestion
  it("creates a HO200117 exception when there are more than 10 recordable results", async () => {
```